File: Plugins/Aspose_Cells_Java_for_Jython/asposecells/WorkingWithWorksheets/ZoomFactor.py
from asposecells import Settings
from com.aspose.cells import Workbook
from com.aspose.cells import SaveFormat


class ZoomFactor:

    def __init__(self):
        dataDir = Settings.dataDir + 'WorkingWithWorksheets/ZoomFactor/'
     
        workbook = Workbook(dataDir + "Book1.xls")

        #Accessing the first worksheet in the Excel file
        worksheets = workbook.getWorksheets()
        worksheet = worksheets.get(0)

        #Setting the zoom factor of the worksheet to 75
        worksheet.setZoom(75)

        #Saving the modified Excel file in default format
        workbook.save(dataDir + "output.xls")

        # Print message
        print "Zoom factor set to 75% for sheet 1, please check the output document."

if __name__ == '__main__':        
    ZoomFactor()

File: Examples/src/main/java/com/aspose/cells/examples/worksheets/ZoomFactor.java
package com.aspose.cells.examples.worksheets;

import com.aspose.cells.*;
import com.aspose.cells.examples.Utils;

public class ZoomFactor {

	public static void main(String[] args) throws Exception {

		// The path to the documents directory.
		String dataDir = Utils.getSharedDataDir(ZoomFactor.class) + "worksheets/";

		// Instantiating a Excel object by excel file path
		Workbook workbook = new Workbook(dataDir + "book1.xls");

		// Accessing the first worksheet in the Excel file
		WorksheetCollection worksheets = workbook.getWorksheets();
		Worksheet worksheet = worksheets.get(0);

		// Setting the zoom factor of the worksheet to 75
		worksheet.setZoom(75);

		// Saving the modified Excel file in default format
		workbook.save(dataDir + "ZoomFactor_out.xls");

		// Print message
		System.out.println("Zoom factor set to 75% for sheet 1, please check the output document.");

	}
}
